config/storage.yml

```diff
@@ -0,0 +1,10 @@
+filesystem:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
+local:
+  service: PostgreSQL
+
+temp_file:
+  service: Disk
+  root: <%= Dir.mktmpdir('test_active_storage') %>
```

Comment:
This means it creates a temporary directory for every test run, but there's no cleanup. Would it make sense to use the tmp dir we already have (`Rails.root.join('tmp'`) as a base?

Reply:
Using rails' tmp folder would be much slower than using `/tmp`, since the latter uses in-memory storage IIRC